### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by N-006**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ The [master](https://github.com/butterproject/butter) branch which contains the
 If you encounter trouble with the above method, you can try:
 
 1. `npm install -g bower grunt-cli` (Linux: you may need to run with `sudo`)
-1. `cd desktop`
```
Comment (MrAdder) [inline README.md]: this is only a possible location it can be anywhere and this is an example location

File: README.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ The [master](https://github.com/butterproject/butter) branch which contains the
...
+1. `cd butter`
 1. `npm install`
 1. `bower install`
 1. `grunt lang`
-1. `grunt nodewebkit`
+1. `grunt nwjs`
```
Comment (MrAdder) [inline README.md]: Will Accept this change

**Comment by MrAdder** [comment]
If you can fix that one change I can merge it

**Comment by N-006** [comment]
Ok, done.

**Pull request merged**